### PR TITLE
Refactoring EP UI components

### PR DESCRIPTION
**Pull request by OleanjiKingCode**
# Refactoring EP UI components

Files Edited:
- About Component / OurHistoryCard : Removed unused <Box> tag
- Activity Component : Aggregating types into a file
- Admin Component Files:  Removed unused tags, Aggregating types into a file, refactored functions, removed unnecessary line spaces
- Blog Component : Aggregating types into a file
-  Branding Component :  Aggregating types into a file
- Categories Component Files : Aggregating types into a file
- Create Wiki Component  Files: Aggregating types into a file, removed unnecessary line spaces
- Glossary Elements Files : Aggregating types into a new  file
- Landing, Wallet Drawer:  Aggregating types into a new  file
- Categories Section on the homepage: Removed the code for getting data from the backend, using the data in the project (AllCategoriesData), also changing the name of the type in all files it was used in.
- Landing and Layout pages 
- Settings pages : new file for their types
- **wallet drawer : removed option to add token to metamask for all except IQ**

closes: https://github.com/EveripediaNetwork/issues/issues/1488

### CODE REVIEW

**Comment by Emmanueldmlr** [comment]
@OleanjiKingCode , there is a conflict, can you fix the conflict first.

**Comment by kesar** [comment]
pls small PRs you can't refactor the whole site in 1 PR because its very hard to review that

**Review by Softdev1 — APPROVED** [review]

**Comment by OleanjiKingCode** [comment]
> pls small PRs you can't refactor the whole site in 1 PR because its very hard to review that

Okay so can i cancel this and make them into small prs? @kesar

**Comment by kesar** [comment]
PR opened for 2 weeks... @Softdev1 pls sync w Emmanuel and get this fixed and merged

**Comment by Softdev1** [comment]
I already reach out to @Emmanueldmlr  to have it reviewed and merge before the close of work today @kesar

**Comment by kesar** [comment]
> I already reach out to @Emmanueldmlr to have it reviewed and merge before the close of work today @kesar

sure, if he is working today in something else its not super urgent. But pls this is something you need to review to avoid this delays and its something you should add in the weekly goals you send me 👍🏻

File: src/config/index.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { env } from '@/env.mjs'
 
 const config = {
   iqAddress: env.NEXT_PUBLIC_IQ_ADDRESS,
+  hiIqAddress: env.NEXT_PUBLIC_HIIQ_ADDRESS,
```
Comment (Emmanueldmlr) [inline src/config/index.ts:5]: Make sure you add this to the env variable on Vercel before merging.

**Review by Emmanueldmlr — APPROVED** [review]

**Comment by Emmanueldmlr** [comment]
@Softdev1, we are good to go. I already tested it to be sure nothing breaks. You should test it also before merging.

File: src/config/index.ts (hunk 1 above)
Comment (Emmanueldmlr) [reply src/config/index.ts:5]: Added already.

**Pull request merged**